src/App.jsx

```diff
@@ -11,7 +11,7 @@ function App() {
     <>
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route path="/Home" element={<Home />} />
+          <Route index path="/Home" element={<Home />} />
```

Comment:
- [ ] Just like the previous reviewer has said, it will be great to make the home page the first thing you see when the site loads. How you have made the change, is not actually working. I suggest you delete this ` <Route path="/" element={<Layout />}>` and on this ` <Route path="/Home" element={<Home />} />` make it to read as so ` <Route path="/" element={<Home />} />`. If you make that change the app will show the homepage once you load